playground/src/tabs-test.vue

```diff
@@ -7,21 +7,21 @@
 
     <!-- Selected Value -->
     <v-tabs
-      data-testid="Selected Test"
+      data-testid="selected-test"
       tabs="Tab 1, Tab 2, Tab 3"
       selected="Tab 2"
     />
 
     <!-- No Selected Value, Default Should Be First Tab -->
     <v-tabs
-      data-testid="Default Test"
+      data-testid="default-test"
       class="mt-2"
       tabs="Tab A, Tab B, Tab C"
     />
 
     <!-- Focus KeyDown Test -->
     <v-tabs
-      data-testid="Key Enter Test"
+      data-testid="key-enter-test"
```

Comment:
[q] is there a reason why you wanted a separate component for the focus keydown test rather than reusing `selected-test`? they seem functionally equivalent to me

Reply:
you are potentially altering the state of the component when clicking a component. So having them separated isn't a bad idea